app/models/setting.rb

```diff
@@ -0,0 +1,14 @@
+require 'bcrypt'
+
+class Setting < ApplicationRecord
+    include BCrypt
+
+    def password
+        @password ||= Password.new(admin_password_hash)
```

Comment:
i'd use `self.admin_password_hash` to be clearer here, as you do in line 12

Reply:
In this case Password.new is necessary since self.admin_password_hash on its own will be the encrypted password saved in the database, but we want @password to be the actual non-hashed password. Password.new(admin_password_hash) will return the non-hashed password we need.